### ISSUE TITLE
Release 2.2.0 is not a "fixed" release

### ISSUE THREAD

**Issue by chisandrei** [pharo-contributions/XML-XPath#1]
It's super to finally see these projects on github!

I was looking at using them and saw that the current release tagged `2.2.0` depends on XML-XMLParser tag `3.5.x`.  I guess tag `3.5.x` is expected to change in the future, for example when release `3.5.1` happens. This means that the `2.2.0` release is not really fixed. 

Now of course any tag could be moved, even `2.2.0`. Just would be better to either have `2.2.0` depend on other fixed versions for its dependencies, or maybe not have `2.2.0` just `2.2.x`

**Comment by astares** [comment]
Yes - the dependencies were choosen to be sliding - but only on the last digit to allow for compatible hotfixes without having to go and reversion the full dependency chain again and again for each. 

The idea of the 3.5.x is to have it for compatible hotfixes or changes that do not affect the release (like changes in documentation or CI). 

If you have a 3.5.1 or 3.5.2, 3.5..3, ... then it is the first, second, third necessary but compatible hotfix / patch on the base version 3.5.0.  

So basically if there is really a new base version (like a milestone) then we update to 3.6.0 with possible hotfixed 3.6.1, 3.6.2, ... again. So you can choose to reference the fixed - but also the sliding. 

If you want to avoid the sliding of the dependencies you can still (re)define to a fixed version in your own baseline.

There are many ways one can do it....
